CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Breaking change]** Now all the modules start with "MSAC" prefix.
```

Comment:
This is not the right wording. Please give some explanation, put placeholder to docs etc